ansible-fmw/fmw_wls_lifecycle.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: Play to start the services of WLS domain
+  hosts: dojoserver
+  become: true
+  become_user: "{{ WLS_USER }}"
+  vars_files:
+    - '{{ playbook_dir }}/wls_variables.yml'
+  tasks:
+    - name: Lifecycle start
+      tags: start
+      block:
```

Comment:
fyi. `nm` should be started before `adminserver` 
and for stop the order is reverse